main.go

```diff
@@ -147,7 +147,8 @@ func fetchAllPrs(c Client, start int, size int, logger *logrus.Logger, prsChanne
 			prsChannel <- v
 		}
 
-		if isLastPage {
+		len := len(prs) // isLastPage is not reliable, so we check for len < size additionally
```

Comment:
I don't think calling the variable `len` is a good idea, because then `len()` will be masked.

Can you find a better name please or simply get rid of the variable and use it directly in the `if` condition?

Thanks!